packages/perseus/src/util/tex.ts

```diff
@@ -64,29 +41,6 @@ export default {
         // Only process if it hasn't been done before, or it is forced
         if ($elem.attr("data-math-formula") == null || force) {
             const $katexHolder = findChildOrAdd($elem, "katex-holder");
-            const $mathjaxHolder = findChildOrAdd($elem, "mathjax-holder");
-
-            // Search for MathJax-y script tags inside of the node. These are
-            // used by MathJax to denote the formula to be typeset. Before, we
-            // would update the formula by updating the contents of the script
-            // tag, which shouldn't happen any more, but we manage them just in
-            // case.
-            const script: HTMLElement | undefined = $mathjaxHolder.find(
-                "script[type='math/tex']",
-            )[0];
-
-            // If text wasn't provided, we look in two places
-            if (text == null) {
-                if ($elem.attr("data-math-formula")) {
-                    // The old typeset formula
-                    // @ts-expect-error - TS2322 - Type 'string | undefined' is not assignable to type 'string'.
-                    text = $elem.attr("data-math-formula");
```

Comment:
hmm, I'm actually not confident we can get rid of this line. `text` is typed as non-null, so I didn't think this code was reachable, but I wonder if the types are telling the truth.